supported_kafka.versions

```diff
@@ -1,3 +1,4 @@
 3.8.0
 3.8.1
-3.9.0
\ No newline at end of file
+3.9.0
+4.0.0
```

Comment:
Out of curiosity (because I don't remember it) - are you following the support matrix which operators do? If so, should 3.8.x be removed?

Reply:
Yeah, I will removed it. Thanks for pointing it out.